extensions/wikia/Email/Controller/WallMessageController.class.php

```diff
@@ -182,7 +182,15 @@ protected function getSubject() {
 	}
 }
 
-class ReplyWallMessageController extends OwnWallMessageController {
+class ReplyWallMessageController extends WallMessageController {
+
+	/** @var \Title */
+	protected $containingThread;
+
+	public function initEmail() {
+		parent::initEmail();
+		$this->containingThread = \Title::newFromText( $this->getVal( 'parentId' ), NS_USER_WALL_MESSAGE );
```

Comment:
`parentId` is already available to us in the request. See [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/WallNotifications/WallNotifications.class.php#L419).